web/api/types/usercredentials.go

```diff
@@ -72,7 +75,9 @@ func (payload *AddUserCredentialRequestPayload) InitFromRequest(r *http.Request)
 	data, err := ioutil.ReadAll(r.Body)
 	gohtypes.PanicIfError("Not possible to parse registration payload", http.StatusBadRequest, err)
```

Comment:
@claudiosegala, could you please replace `registration payload` to `update payload` here? I think this is a previou unknown feedback bug.

Reply:
But isn't the function from AddUserCredential? Why `update payload`?